src/josepy/jws.py

```diff
@@ -379,25 +391,25 @@ def verify(cls, args):
         else:
             key = None
 
-        sys.stdout.write(sig.payload)
+        sys.stdout.write(sig.payload.decode())
         return not sig.verify(key=key)
 
     @classmethod
-    def _alg_type(cls, arg):
+    def _alg_type(cls, arg: Any) -> jwa.JWASignature:
         return jwa.JWASignature.from_json(arg)
 
     @classmethod
-    def _header_type(cls, arg):
+    def _header_type(cls, arg: Any) -> Any:
         assert arg in Signature.header_cls._fields
         return arg
 
     @classmethod
-    def _kty_type(cls, arg):
+    def _kty_type(cls, arg: Any) -> Type[jwk.JWK]:
         assert arg in jwk.JWK.TYPES
         return jwk.JWK.TYPES[arg]
 
     @classmethod
-    def run(cls, args=None):
+    def run(cls, args: List[str] = None) -> Any:
```

Comment:
Maybe
```suggestion
    def run(cls, args: List[str] = None) -> Optional[bool]
```
if the above suggestion is also taken?

I think we should probably rip out all of this code long term though.

Reply:
Also done.